omdv-build-iso.sh

```diff
@@ -1249,6 +1250,10 @@ setupGrub2() {
 	    errorCatch
 	fi
     fi
+# Fix up 2014.0 grub installer line...We don't have Calamares in 2014.
+    if [ $VERSION == OPENMANDRIVA2014.0 ]; then
```

Comment:
@itchka  Please when do comparin do it on lowercase... keep consistency with rest of the code.

${version,,} =~ openmandriva201.*